main.py

```diff
@@ -1 +1,17 @@
 #Mad Libs Generator Project
+verb_1 = input("Enter a verb of choice, and press enter:")
+adj_1 = input("Enter a adjective of choice, and press enter:")
+verb_2 = input("Enter second verb of choice, and press enter:")
+body_part = input("Enter a body part name and press enter:")
```

Comment:
Consider consistency of the prompt with surrounding prompts.

For instance
```suggestion
body_part = input("Enter a body part name, and press enter:")
```